src/interfaces/account-state.interface.ts

```diff
@@ -14,18 +14,7 @@ export interface AccountStateInterface {
 export const initialAccountState: AccountStateInterface = {
   isVisible: true,
   tezosBalance: '0',
-  tokensList: [
-    ...MAINNET_TOKENS_METADATA.map(token => ({
-      slug: getTokenSlug(token),
-      balance: '0',
-      visibility: VisibilityEnum.Visible
-    })),
-    ...HIDDEN_WHITELIST_TOKENS.map(token => ({
-      slug: getTokenSlug(token),
-      balance: '0',
-      visibility: VisibilityEnum.InitiallyHidden
-    }))
-  ],
+  tokensList: [],
```

Comment:
I think we still need some `INITIAL_WHITELIST_TOKENS` list  
let's discuss this on deily